.github/workflows/ci.yml

```diff
@@ -51,8 +51,8 @@ jobs:
         # Keep in sync with the version in GitLabDockerContainer.cs
         # Available tags: https://hub.docker.com/r/gitlab/gitlab-ee/tags
         gitlab: [
-          'gitlab/gitlab-ee:15.11.9-ee.0',
-          'gitlab/gitlab-ee:15.11.13-ee.0',
+          'gitlab/gitlab-ee:16.6.2-ee.0',
```

Comment:
Ideally we would also keep on supporting 15.11.*.

Reply:
[70b831d](https://github.com/ubisoft/NGitLab/pull/682/commits/70b831d39b2eb5dbbfe74cbfa903397ec16b85e5)